Gemfile.lock

```diff
@@ -1,6 +1,6 @@
 GIT
   remote: https://github.com/ideacrew/aca_entities.git
-  revision: 87b6170325d9040bc8a327481dda5cfcc8501514
+  revision: 5471a2d06bba7fc784f93403a1b807ef88b9f3b7
```

Comment:
Should this be in it's own PR again, or was that just for enroll?

Reply:
this one breaks compatibility, its not just refactor, we cannot push sha alone because the test will break